data-science-pipelines-operator/base/params.env

```diff
@@ -0,0 +1,10 @@
+IMAGES_APISERVER=quay.io/modh/odh-ml-pipelines-api-server-container
+IMAGES_ARTIFACT=quay.io/modh/odh-ml-pipelines-artifact-manager-container
+IMAGES_PERSISTENTAGENT=quay.io/modh/odh-ml-pipelines-persistenceagent-container
+IMAGES_SCHEDULEDWORKFLOW=quay.io/modh/odh-ml-pipelines-scheduledworkflow-container
+IMAGES_VIEWERCRD=quay.io/modh/odh-ml-pipelines-viewercontroller-container
+IMAGES_CACHE=registry.access.redhat.com/ubi8/ubi-minimal
+IMAGES_MOVERESULTSIMAGE=registry.access.redhat.com/ubi8/ubi-micro
```

Comment:
@gmfrasca / @rimolive  can we get the latest images built for 1.24 so I can add them here?